apps/portals/index.html

```diff
@@ -2,40 +2,31 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.svg" />
+    <link rel="shortcut icon" href="/favicon.svg" />
 
     <!-- Facebook Meta Tags -->
-    <meta property="og:url" content="%PRODUCTION_URL%" />
+    <meta property="og:url" content="https://%VITE_PORTAL_KEY%.synapse.org/" />
```

Comment:
Prior to this change, we had a Python script that parsed a portal-specific JSON file and replaced these strings in the index.html at build-time.

Now, Vite pulls from a portal-specific `.env` file and replaces these when serving (devmode) or building.

So this is now one less step we maintain in our build process, and one less difference between development and production environments.